modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/scim2/rest/api/SCIMUtils.java

```diff
@@ -102,4 +108,45 @@ public static boolean isValidSCIMDate(String date) {
         }
         return true;
     }
+
+    /**
+     * Create a secondary user store.
+     *
+     * @param userStoreType       User store type.
+     * @param userStoreDomain     User store domain
```

Comment:
Missing full stop.

Reply:
Resolved in https://github.com/wso2/product-is/pull/7439/commits/5f62c85aab90f9520469d3a7da2fb302dbdfcc27